---
permalink: /
title: ""
author_profile: true
redirect_from: 
  - /about/
  - /about.html
---
Research interests 
======
Working in the interface of genomics, single cell, deep learning and big data analysis with my PhD advisor Prof. [Qiangfeng Cliff Zhang](https://life.tsinghua.edu.cn/lifeen/info/1034/1075.htm), my main research interest is to develop and apply innovative machine learning methods for the studies of gene regulation and cell fate transition on the basis of advanced single cell techniques (e.g., spatial transcriptomics, single cell RNA-seq, and single cell multi-omics). The long-term goal of my work is to decipher the gene regulation of the human genome and its impact on cell state transition for disease diagnosis and therapy. 

Feel free to reach out by email liyuzhe at pku.edu.cn.

Research experience 
======
Graduate Research: 
1.	We developed SCALEX, a deep-learning method that integrates single cell data by projecting cells into a batch-invariant, common cell-embedding space in a truly online manner (i.e., without retraining the model). 
2.	We characterized a novel role for smarca5 in red blood cell (RBC) aggregation using zebrafish smarca5 mutants, which may provide a new venous thrombosis animal model to support drug screening and pre-clinical therapeutic assessments to treat thrombosis. I'm in charge of data analysis, mainly for RNA-seq and ATAC-seq data.
3.	We developed SPACE, a deep-learning method for cell type identification and tissue module discovery from spatial transcriptomics data by learning a cell representation that captures its gene expression profile and those of its spatial neighbors.
4.	We developed a deep learning-based model, SCAPE, that predicts the transcription dynamics under in silico genetic perturbations. The manuscript is in preparation.
5.	My current graduate research focused on deciphering gene regulatory networks for cells in human diseases (e.g., pediatric brain tumor, lung cancer, and idiopathic pulmonary fibrosis) using single cell multi-omics. The manuscripts are in preparation.

Current Research and Scholarly Interests
======
My research focuses on develop deep learning methods to
1. Spatial multi-omics data analysis
2. Infer the dynamics of cell state transition
3. Build foundational model for single-cell genomics
