---
title: "Online single-cell data integration through projecting heterogeneous datasets into a common cell-embedding space."
collection: publications
permalink: /publication/2022-paper-title-number-2
excerpt: ''
date: 2022-10-17
venue: 'nature communications '
paperurl: 'https://www.nature.com/articles/s41467-022-33758-z'
citation: 'Xiong, L.*, Tian, K.*, Li, Y., Ning, W., Gao, X., and Zhang, Q.C. Online single-cell data integration through projecting heterogeneous datasets into a common cell-embedding space. Nat Commun 13, 1–17 (2022).'
---

Computational tools for integrative analyses of diverse single-cell experiments are facing formidable new challenges including dramatic increases in data scale, sample heterogeneity, and the need to informatively cross-reference new data with foundational datasets. Here, we present SCALEX, a deep-learning method that integrates single-cell data by projecting cells into a batch-invariant, common cell-embedding space in a truly online manner (i.e., without retraining the model). SCALEX substantially outperforms online iNMF and other state-of-the-art non-online integration methods on benchmark single-cell datasets of diverse modalities, (e.g., single-cell RNA sequencing, scRNA-seq, single-cell assay for transposase-accessible chromatin use sequencing, scATAC-seq), especially for datasets with partial overlaps, accurately aligning similar cell populations while retaining true biological differences. We showcase SCALEX’s advantages by constructing continuously expandable single-cell atlases for human, mouse, and COVID-19 patients, each assembled from diverse data sources and growing with every new data. The online data integration capacity and superior performance makes SCALEX particularly appropriate for large-scale single-cell applications to build upon previous scientific insights.
