---
title: "Tissue module discovery in single-cell resolution spatial transcriptomics data via cell-cell interaction-aware cell embedding."
collection: publications
permalink: /publication/2024-paper-title-number-1
excerpt: ''
date: 2024-05-31
venue: 'Cell Systems'
paperurl: 'https://www.cell.com/cell-systems/fulltext/S2405-4712(24)00124-8'
citation: 'Li, Y.*, Zhang J.*, Gao, X., and Zhang, Q.C. Tissue module discovery in single-cell resolution spatial transcriptomics data via cell-cell interaction-aware cell embedding. Cell Syst (2024) May 29: S2405-4712(24)00124-8.'
---

Computational methods are desired for single-cell-resolution spatial transcriptomics (ST) data analysis to uncover spatial organization principles for how individual cells exert tissue-specific functions. Here, we present ST data analysis via interaction-aware cell embedding (SPACE), a deep-learning method for cell-type identification and tissue module discovery from single-cell-resolution ST data by learning a cell representation that captures its gene expression profile and interactions with its spatial neighbors. SPACE identified spatially informed cell subtypes defined by their special spatial distribution patterns and distinct proximal-interacting cell types. SPACE also automatically discovered “cell communities”—tissue modules with discernible boundaries and a uniform spatial distribution of constituent cell types. For each cell community, SPACE outputs a characteristic proximal cell-cell interaction network associated with physiological processes, which can be used to refine ligand-receptor-based intercellular signaling analyses. We envision that SPACE can be used in large-scale ST projects to understand how proximal cell-cell interactions contribute to emergent biological functions within cell communities. 
